index.html

```diff
@@ -1806,30 +1809,49 @@
                   </ol>
                 </li>
                 <li>If <var>previousAvailability</var> is not equal to
-                <var>newAvailability</var>, then <a>queue a task</a> to run the
-                following steps:
+                <var>newAvailability</var>, then run the following steps:
                   <ol>
-                    <li>Set <var>A</var>'s <a>value</a> property to
-                    <var>newAvailability</var>.
+                    <li>If <var>A</var> has already been exposed to ECMAScript
```

Comment:
I would like to not rely on "exposed to ECMAscript" to define behavior since (as an implementer) I'm not entirely sure what that means.

Here are two thoughts:

- The first time the monitoring algorithm runs for A, the promise for A should be unsettled, correct?  Could you fire the event only after the availability promise is resolved?

- Could step 8 of getAvailability initialize `value` to `undefined`, and then the event would fire only when the previous value was set?